frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -495,6 +515,29 @@ const plugin: Plugin<ConsumedExtensions> = [
         )).default,
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      exact: true,
+      path: ['/dev-monitoring/all-namespaces'],
+      loader: async () =>
+        (await import(
+          './components/monitoring/MonitoringPage' /* webpackChunkName: "dev-console-monitoring" */
+        )).default,
+    },
+  },
+  {
+    type: 'Page/Route',
+    properties: {
+      perspective: 'dev',
+      path: ['/dev-monitoring/ns/:ns'],
+      loader: async () =>
+        (await import(
+          './components/monitoring/MonitoringPage' /* webpackChunkName: "dev-console-monitoring" */
+        )).default,
+    },
+  },
```

Comment:
I don't think this is appropriate. Two different routes configs to the same component? I think we should either do one for the "all project" list and one for the "Details" or just house them both under the non-exact config you had before. This is not helping much.

Perhaps @christianvogt knows of a better use of the plugin to get around the previous issue you had @abhi-kn - would recommend reaching out to him tomorrow.

Reply:
Technically this is correct.
But I *think* you can create a single path as follows with `exact: false`:
```
/dev-monitoring(/all-namespaces$|/ns/:ns)
```

Reply:
Tried with suggested path & it works partially. Basically `:ns` doesn't work so I've reverted to single route with multiple paths.